infra/src/main/java/com/redhat/service/smartevents/infra/models/dto/BridgeStatusWrapperDTO.java

```diff
@@ -0,0 +1,63 @@
+package com.redhat.service.smartevents.infra.models.dto;
+
+import java.util.Objects;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.service.smartevents.infra.exceptions.BridgeError;
+
+public class BridgeStatusWrapperDTO {
```

Comment:
What about adding the error code in the `BridgeDTO` in case of failure? In that case we might simply return the error code in the response and then it's up to the user to look up the error catalog for more details